dateparser/date.py

```diff
@@ -120,11 +122,18 @@ def get_date_from_timestamp(date_string, settings, negative=False):
          match = RE_SEARCH_TIMESTAMP.search(date_string)
 
     if match:
+        if settings is not None and settings.TIMEZONE is not None and 'local' not in settings.TIMEZONE.lower():
+            local_timezone = get_timezone_from_tz_string(settings.TIMEZONE)
+        else:
+            local_timezone = get_localzone()
```

Comment:
Will it make more sense to use the variable as `timezone` or `current_timezone` instead of `local_timezone` as the timezone coming from `get_timezone_from_tz_string(settings.TIMEZONE)` may not be the local timezone?

Reply:
yeah that makes sense